recipe/meta.yaml

```diff
@@ -252,7 +252,7 @@ outputs:
         - ciso8601 >=2.3.1
         - cloudpickle >=3.0.0
         - cookiecutter >=2.6.0
-        - dask-core >=2024.5.0
+        - dask-core >=2024.5.0,<2025.1.0  # Unpin once datashader 0.17.0
```

Comment:
Need to remember to apply that to rubinenv 10 as well.

Reply:
Is that the dev branch?

Reply:
#285

Reply:
No, that's ancient.  Where do I find rubenv10 ?

Reply:
That's the only rubin-env 10 PR we have. It hasn't merged to dev yet but should be soon.

Reply:
OK, I've PRed this change against the roceb:dev branch as well.  (Also he forked it before we dropped dash, so I fixed that too).